### PR TITLE
Added additionalTabClass for Tab Class

### PR DESCRIPTION
**Pull request by cliedelt**
We wanted to add a CSS-Class for a single Tab.

### CODE REVIEW

**Comment by brrd** [comment]
```javascript
tabGroup.addTab({
  ready: ({tab}) => tab.classList.add("classname");
});
```

Doesn't this already do the trick?

File: README.md [hunk 1]
```diff
@@ -101,6 +101,7 @@ Add a new tab to the tab group and returns a `Tab` instance.
 * `visible` (default: `true`): set this to `false` if you don't want to display the tab once it is loaded. If set to `false` then you will need to call `tab.show()` to display the tab.
 * `active` (default: `false`): set this to `true` if you want to activate the tab once it is loaded. Otherwise you will need to call `tab.activate()`.
 * `ready`: a callback function to call once the tab is ready. The `Tab` instance is passed as the only parameter.
+*  `additionalTabClass`: optional css class which is added to a single tab.
```
Comment (MCMicS) [inline README.md:104]: two spaces instead one

File: index.d.ts [hunk 2]
```diff
@@ -39,6 +39,7 @@ declare namespace ElectronTabs {
     webviewAttributes?: {[key: string]: any};
     visible?: boolean;
     active?: boolean;
+    additionalTabClass?: string;
```
Comment (MCMicS) [inline index.d.ts:42]: it is TabOption so it should be enough to name it: `additionalClass`

**Comment by MCMicS** [comment]
> ```js
> tabGroup.addTab({
>   ready: ({tab}) => tab.classList.add("classname");
> });
> ```
> 
> Doesn't this already do the trick?

Yes this is working too.

**Comment by brrd** [comment]
> > ```js
> > tabGroup.addTab({
> >   ready: ({tab}) => tab.classList.add("classname");
> > });
> > ```
> > 
> > Doesn't this already do the trick?
> 
> Yes this is working too.

So we obviously don't need this PR.

**Pull request closed without merging**